template/javascript/base/src/components/HelloWorld.vue

```diff
@@ -1,32 +1,25 @@
 <template>
-  <v-container class="fill-height">
-    <v-responsive
-      class="align-centerfill-height mx-auto"
-      max-width="900"
-    >
+  <v-container class="fill-height" width="900">
+    <div>
       <v-img
-        class="mb-4"
         height="150"
-        src="@/assets/logo.png"
+        src="~/assets/logo.png"
```

Comment:
This changes breaks things for me. Used pnpm as package manager.

Reply:
I have basically copied Nuxt version to other places and overlooked that was the difference between versions.